### DISCUSSION TITLE
Intent Filter Verification Service always crashes

### DISCUSSION THREAD

**Discussion by d4rklynk** [Divested-Mobile/DivestOS-Build#112] [Q&A] [answered]
**Stuff I have :** 
 - OnePlus 6T (fajita)
 - DivestOS version : 19-20220511-dos-fajita
 - Bootloader locked

**What's happening :** 
 - I don't know tbh, it crashes randomly whenever I do whatever
 - It is happening like once an hour en even every 3~4 minutes sometimes

**A proper screenshot :**
![Screenshot_20220521-012501_Aurora Store](https://user-images.githubusercontent.com/74207682/169645508-21b1e7e0-9892-4e34-bbbf-619fb640fa4d.png)

**Comment by SkewedZeppelin** [comment]
I haven't seen this, can you provide a logcat?

`adb logcat -b all -d *:E`

**Reply by d4rklynk** [reply]
Obviously, everything work for now, I'll do a log when the crash will appear again

**Reply by d4rklynk** [reply]
I'll give you this log (I just grep all the com.android.statementservice.) But I didn't have any crash today
[com.android.statementservice.txt](https://github.com/Divested-Mobile/DivestOS-Build/files/8747968/com.android.statementservice.txt)

**Reply by d4rklynk** [reply]
The bug appeared again here's the log this time
[log deleted for privacy reasons]

**Reply by SkewedZeppelin** [reply]
StatementService appears to be crashing because it puts too much data in a queue.
The problematic app causing it isn't shown in the error.

You should be able to safely disable StatementService via Settings/App Info.

**Reply by d4rklynk** [reply]
I can't disable the app, but I can disable all permissions though

**Reply by SkewedZeppelin** [reply]
That'd likely make things worse.
Perhaps clear the app data for it for now and see if it happens again?

**Reply by d4rklynk** [reply]
I already did a few times

**Comment by SkewedZeppelin** [comment, marked as answer]
This error message is suppressed in recent versions, until the bug is fixed (hopefully upstream).
